libminifi/test/rocksdb-tests/DBProvenanceRepositoryTests.cpp

```diff
@@ -103,8 +103,6 @@ TEST_CASE("Test time limit", "[timeLimitTest]") {
 
   provisionRepo(provdb, keyCount / 2, 102400);
 
-  REQUIRE(provdb.getKeyCount() == 250);
-
   /**
```

Comment:
What happened here? Can you modify the comment as well why we dont have 250 entries?

Reply:
I think the comment is okay as we do put 250 entries to the DB. The problem here was that sometimes when we get to this point of the execution the key count is already 96 instead of 250, as the serialized records have probably already expired.